contracts/staking/SimpleStaking.sol

```diff
@@ -262,12 +271,16 @@ contract SimpleStaking is AbstractGoodStaking, StakingToken {
 			_recipient != address(this),
 			"Recipient cannot be the staking contract"
 		);
+		address daiAddress = nameService.getAddress("DAI");
 		(uint256 iTokenGains, uint256 tokenGains, uint256 precisionLossToken) =
 			currentUBIInterest();
 		lastUBICollection = block.number.div(blockInterval);
-		if (iTokenGains > 0)
+		(address redeemedToken, uint256 redeemedAmount) =
+			redeemUnderlying(iTokenGains);
+		require(redeemedToken == daiAddress, "Redeemed token should be DAI");
```

Comment:
no need to enforce it.
we might also want to accept cDAI.
this will be enforced by the DAO when voting in

Reply:
But I should check it if cDAI or DAI?